lib/WriteVm/test/readLog.rb

```diff
@@ -4,7 +4,7 @@
 
 require 'rubygems'
 require 'log4r'
-require 'VimClientBase'
+require 'MiqVimClientBase'
 require 'MiqVim'
```

Comment:
Requiring MiqVim causes MiqVimClientBase to be loaded, so there should be no need to require it here.